js/controller.js

```diff
@@ -631,28 +642,7 @@ function controller(OO, _, $) {
         videoElement.addEventListener('loadedmetadata', this.metaDataLoaded.bind(this));
 
         // add the AirPlay event listeners
-        if (window.WebKitPlaybackTargetAvailabilityEvent) {
-          const airPlayState = window.sessionStorage.getItem('airPlayState');
-          if (airPlayState === CONSTANTS.AIRPLAY_STATE.CONNECTED || this.state.isAirplayAllowed) {
-            videoElement.addEventListener(
-              'webkitplaybacktargetavailabilitychanged',
-              _.bind(this.airPlayListener, this)
-            );
-
-            // This event fires when a media element starts or stops AirPlay playback
-            videoElement.addEventListener(
-              'webkitcurrentplaybacktargetiswirelesschanged',
-              _.bind(this.toggleAirPlayIcon, this)
-            );
-
-            // Showing the AirPlay target picker if it was connected before reloading
-            this.showAirplayPickerBound = this.showAirplayPicker.bind(this);
-            videoElement.addEventListener(
```

Comment:
are you sure to remove this listener?

Reply:
I have moved this functionality to the `toggleAirPlayIcon`, because we should call the target picker after the second event of the `webkitcurrentplaybacktargetiswirelesschanged`